app/controllers/messages_controller.rb

```diff
@@ -92,10 +95,16 @@ def create_pusher_payload(new_message, temp_id)
       timestamp: Time.current,
       color: new_message.preferred_user_color,
       reception_method: "pushed"
-    }.to_json
+    }
+
+    if new_message.chat_channel.channel_type != "direct"
```

Comment:
see my above note. 

ps. why do we need to adjust the slug in case of a group channel? maybe you can add the reason why as a comment

Reply:
As I need the channel adjusted slug to generate a group mention notification.